src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -287,6 +287,20 @@ public SuiteResult getPreviousResult() {
         return null;
     }
 
+     /**
+     * Check if all the tests exists for the suiteResult
+     * @param suiteResult
+     * @return true if all the tests were found, false otherwise
+     */
+    public boolean includesTestsFrom(SuiteResult suiteResult) {
```

Comment:
Laso it makes sense to add <code>@since TODO</code> to document method in API when it gets merged